Chatto/Source/ChatController/BaseChatViewController.swift

```diff
@@ -233,21 +237,25 @@ open class BaseChatViewController: UIViewController, UICollectionViewDataSource,
         }
     }
 
-    private func adjustCollectionViewInsets() {
+    public var allContentFits: Bool {
+        let inputHeightWithKeyboard = self.view.bounds.height - self.inputContainer.frame.minY
+        let newInsetTop = self.topLayoutGuide.length + self.layoutConfiguration.contentInsets.top
```

Comment:
What does that mean "new"? Do you change insets here? Can you change names to`insetTop` and `insetBottom`?